tern/classes/docker_image.py

```diff
@@ -130,11 +130,17 @@ def set_layer_created_by(self):
                 else:
                     self._layers[index].created_by = ''
                 index = index + 1
+                if index is self.until_layer:
+                    break
 
-    def load_image(self):
+    def load_image(self, until_layer=0):
         """Load metadata from an extracted docker image. This assumes the
         image has already been downloaded and extracted into the working
         directory"""
+        if until_layer is None:
```

Comment:
Not sure we need this check -- `until_layer` shouldn't be `None` at this point. If a layer is not provided by the user it should default to 0 and we can do this through the argument parser with default=0.